ngrinder-core/src/main/java/net/grinder/util/AbstractGrinderClassPathProcessor.java

```diff
@@ -224,22 +227,41 @@ public String buildForemostClasspathBasedOnCurrentClassLoader(Logger logger) {
 	 * @return classpath optimized for grinder.
 	 */
 	public String buildPatchClasspathBasedOnCurrentClassLoader(Logger logger) {
-		URL[] urLs = ((URLClassLoader) AbstractGrinderClassPathProcessor.class.getClassLoader()).getURLs();
+		URL[] urls = urlsFromClassLoader(AbstractGrinderClassPathProcessor.class.getClassLoader());
 		StringBuilder builder = new StringBuilder();
-		for (URL each : urLs) {
+		for (URL each : urls) {
 			builder.append(decodePathWithUTF8(each.getFile())).append(File.pathSeparator);
 		}
 		return filterPatchClassPath(builder.toString(), logger);
 	}
 
+	public static URL[] urlsFromClassLoader(ClassLoader classLoader) {
```

Comment:
is it the appropriate method name?

I think it's better to name it like...
getClassPaths?

Reply:
@junoyoon 

Renamed the method. and I removed all the codes that cast class loader to `URLClassLoader`.